src/main/java/at/hannibal2/skyhanni/api/SkillApi.kt

```diff
@@ -62,23 +62,30 @@ object SkillApi {
 
     // TODO find out whats going on here
     /**
-     * REGEX-TEST: Farming 35: §r§a12.4%
+     * REGEX-TEST:  Farming 35: §r§a12.4%
+     * REGEX-TEST:  §r§aFarming 35: §r§a12.4%
```

Comment:
did you invent this regex test or is this something you got ingame

Reply:
I assumed that this also can be possible